components/suivi-form/livrables/livrable-card.js

```diff
@@ -117,6 +121,7 @@ LivrableCard.propTypes = {
     licence: PropTypes.string.isRequired,
     diffusion: PropTypes.string,
     avancement: PropTypes.number,
+    stockage: PropTypes.string,
```

Comment:
On peut préciser les types de stockage.
```suggestion
    stockage: PropTypes.oneOf(['http', 'ftp']),
```

Et bientôt `sftp` du coup.